toolbox/Dockerfile

```diff
@@ -24,10 +23,19 @@ RUN apt-get update && \
         fio \
         gdb \
         iperf3 \
+        jq \
         less \
         man-db \
         vim \
     && apt-get clean && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
 
+ENV ROOK_VERSION v0.2.2
```

Comment:
per our conversation i think we should pick up the local build rather than pulling a specific build version